src_types/common/Buffer.d.ts

```diff
@@ -0,0 +1,2 @@
+declare class Buffer extends Uint8Array{
```

Comment:
it's not really Buffer, maybe we should rename the file

Reply:
I also, change it to an empty class(not extending any thing)